lib/minify.js

```diff
@@ -10,6 +10,10 @@ exports.simple_glob = function(files) {
     return files;
 };
 
+var defaultBase64Decoder = exports.base64Decoder = function(input) {
```

Comment:
Actually, lines 13 - 15 can be removed altogether and this base64 decode function definition can replace the one below.